src/assets/icons/MoveTo.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M5.509 1C5.78017 1 6.15694 1.15694 6.34837 1.34837L7.65163 2.65163C7.84403 2.84403 8.2308 3 8.50044 3H15.003C15.5536 3 16 3.45576 16 4.00247V12.9915C16 14.1007 15.1055 15 14.0059 15H1.99406C0.892771 15 0 14.1029 0 12.9941V2.00685C0 1.45078 0.450975 1 0.990778 1H5.509ZM11.5 6C11.1095 5.60948 10.4763 5.60948 10.0858 6C9.7253 6.36048 9.69757 6.92771 10.0026 7.32001L10.0858 7.41421L10.5858 8H6C5.44772 8 5 8.44771 5 9C5 9.51284 5.38604 9.93551 5.88338 9.99327L6 10H10.5858L10.0858 10.5858C9.69526 10.9763 9.69526 11.6095 10.0858 12C10.4463 12.3605 11.0135 12.3882 11.4058 12.0832L11.5 12L13.7071 9.70711C14.0676 9.34662 14.0953 8.77939 13.7903 8.3871L13.7071 8.29289L11.5 6Z" fill="#1D212A" fill-opacity="0.72"/>
+</svg>
```

Comment:
@joel-costa Should we remove the fill and opacity attribute? How do we display it in other context with other colors configuration ? 

Also should it be added to UI ? Should we use it somewhere else in Cozy ?

Reply:
+1 pour les fill, voir la doc de cozy-ui pour l'ajout d'icone.

Reply:
+1 pour les fill, voir la doc de cozy-ui pour l'ajout d'icone.

Reply:
+1 to remove the fill ans opacity attribute

Currently it is not used somewhere else in Cozy but I think it will be in the future, probably in a connector to change the destination folder

Reply:
Ok, icon added with this PR: https://github.com/cozy/cozy-ui/pull/2027